mk_verificator/tests/system/test_pkg_version.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from mk_verificator import utils
+
+
+@pytest.mark.parametrize(
+    "group",
+    utils.get_groups(utils.get_configuration(__file__)))
+def test_pkg_version(local_salt_client, group):
+
+    pkgs_info = local_salt_client.cmd(group, 'lowpkg.list_pkgs')
+
+    groups = {}
+
+    missed_pkgs, version_conflicts = [], []
+
+    for node_name, node_pkgs in pkgs_info.items():
+        group_name = node_name.split('-')[0]
+        if not groups.has_key(group_name):
+            groups[group_name] = [(node_name, node_pkgs)]
+        else:
+            groups[group_name].append((node_name, node_pkgs))
```

Comment:
This part is unnecessary
utils.get_groups returns groups.
For example: [cpu*, ctl*, ...]